test/e2e/portforward.go

```diff
@@ -195,7 +187,16 @@ var _ = Describe("Port forwarding", func() {
 				Failf("Expected %q from server, got %q", e, a)
 			}
 
-			logOutput := runKubectlWithTimeout(wait.ForeverTestTimeout, "logs", fmt.Sprintf("--namespace=%v", framework.Namespace.Name), "-f", podName)
+			By("Waiting for the target pod to stop running")
+			framework.WaitForPodNoLongerRunning(pod.Name)
+
+			By("Retrieving logs from the target pod")
+			logOutput, err := getPodLogs(framework.Client, framework.Namespace.Name, pod.Name, "portforwardtester")
```

Comment:
No, it doesn't, but the point of these tests is to test that `kubectl port-forward` works. I don't think how the logs are retrieved really matters... do you?